app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+
+class UserController extends Controller
+{
+	const ROLES = 'enum.user_roles';
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+		$users = User::where('role', config(self::ROLES)['sys_admin'])->get();
+		return $users;
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('admin.users.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+		$user = User::create($request->all());
```

Comment:
Maybe seria conveniente agregar validación de los campos en back-end, just in case? Si quieres checa como lo puse en `CourseController` para darte una idea.